packages/block-library/src/columns/edit.native.js

```diff
@@ -148,32 +158,50 @@ function ColumnsEditContainer( {
 		return null;
 	};
 
-	const getColumnsSliders = () => {
-		const columnWidths = innerColumns.map(
-			( innerColumn ) =>
-				parseFloat( innerColumn.attributes.width ) || 100 / columnCount
+	const onChangeWidth = ( nextWidth, valueUnit, columnId ) => {
+		nextWidth = 0 > parseFloat( nextWidth ) ? '0' : nextWidth;
```

Comment:
It looks the same as https://github.com/WordPress/gutenberg/pull/26161/files#diff-0a31b6b4b1dea0b86812a4333da8edd1ff843026c6d59fa14453ebd6ff5a4130R64

Have you considered to move it to some util function that takes 2 args (nextWidth, valueUnit) and returns string  `${ nextWidth }${ valueUnit }`?